src/config/config.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+
+module.exports = {
+  development: {
+    username: 'rostammahabadi',
+    password: null,
+    database: 'initiateco',
+    host: '127.0.0.1',
+    dialect: 'postgresql',
+    seederStorage: "sequelize"
+  },
+  production: {
```

Comment:
Great use of environment variables so our information isn't exposed. Bless.

Reply:
Very very nice!